src/test/java/GmailActionableEmailsServletTest.java

```diff
@@ -37,16 +41,42 @@
 
   private static final Gson gson = new Gson();
 
-  // Spaces should be parsed correctly by request.getParameter if query string is properly encoded
-  private static final String SUBJECT_LINE_PHRASES_STRING = "Action Word One, ActionWordTwo";
+  // Spaces within words are fine (and would be encoded/decoded by request.getParameter)
+  private static final String SUBJECT_LINE_PHRASES_STRING = "\"Action Word One\",\"ActionWordTwo\"";
+  private static final List<String> SUBJECT_LINE_PHRASES_LIST =
+      Arrays.asList("\"Action Word One\"", "\"ActionWordTwo\"");
```

Comment:
Are you assuming that the client will always add quotes around the phrases? Does it even matter here?

I think you are handling the quotes in the GmailClientImpl (ignoring/adding them). Maybe just assume the inputs don't contain quotes, and don't send quotes in your actual client. It just complicates things and makes it seem like the quotes are significant. (If the user enters quotes, then they'll be ignored, and that's fine.)

Reply:
Yes fair enough - I feel extra safe making this assumption here since the quotes handling is done by the GmailClient (and it is tested). 

Changing!